### PR TITLE
Blog app - controllers specs

### PR DESCRIPTION
**Pull request by harshitajain06**
- Wrote tests for controllers.

### CODE REVIEW

File: spec/requests/posts_spec.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      get '/users/:user_id/posts'
+      expect(response).to render_template(:index)
+    end
+
+    it 'includes correct placeholder text in the response body' do
+      get '/users/:user_id/posts'
+      expect(response.body).to include('Here is a list of posts for a given user')
+    end
+  end
+end
```
Comment (ShadyShawkat) [inline spec/requests/posts_spec.rb]: - [ ] Please make sure to test for all controllers including the `Posts#show` controller. As per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=Create%20a%20Request%20spec%20file%20for%20all%20your%20controllers.) requirement.

File: spec/requests/users_spec.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      get '/users'
+      expect(response).to render_template(:index)
+    end
+
+    it 'includes correct placeholder text in the response body' do
+      get '/users'
+      expect(response.body).to include('All the users')
+    end
+  end
+end
```
Comment (ShadyShawkat) [inline spec/requests/users_spec.rb]: - [ ] Please make sure to test for all controllers including the `Users#show` controller. As per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=Create%20a%20Request%20spec%20file%20for%20all%20your%20controllers.) requirement.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @harshitajain06, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MaganaAsiati — APPROVED** [review]
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @harshitajain06  , I'm @MaganaAsiati , your reviewer for this PR  :wave: 
Great work on making the changes requested by a previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 





## Highlights
- Descriptive pull request :heavy_check_mark:👨🏻‍💻
- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future.
- [ ] Check the comments under the review.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
Cheers and Happy coding! 🤓 👏👏👏

**Pull request merged**